ui/react-components/components/CancelConfirmation/CancelConfirmation.module.scss

```diff
@@ -3,7 +3,10 @@
 .cancelModal {
   .cancelModalCloseIcon {
     text-align: right;
-    a{
+    button{
```

Comment:
do we need to handle style of button on focus? What is the expected on focus style?

Reply:
it is not for focus, earlier we used anchor tag for icon, but now for tab functionality  we need button in place of anchor tag, so we need to change the css of button for it to look like normal icon

Reply:
As we have already mentioned the styles for the button, if we need separate styles other than already existing then we write separate styles for the focus, else the same will reflect for normal and focus state. So we dont think we need to write separate css for focus